File: src/Client/inventory-track/src/app/models/dto/warehouse/update-warehouse-dto.ts
// src/app/models/dto/warehouse/update-warehouse-dto.ts
import {WarehouseType} from "./enums/warehouse-type.enum";

export interface UpdateWarehouseDto {
  id: string;
  name: string;
  type: number; // WarehouseType в строковом формате
  location: string;
  companyId: string;
  responsiblePersonId: string;
}

File: src/Services/InventoryService/InventoryService.Application/DTOs/Request/Warehouse/UpdateWarehouseDto.cs
using InventoryService.Domain.Enums;

namespace InventoryService.Application.DTOs.Request.Warehouse;

public class UpdateWarehouseDto
{
    public Guid Id { get; set; }
    public string Name { get; set; }
    public WarehouseType Type { get; set; }
    public string Location { get; set; }
    public Guid CompanyId { get; set; }
    public Guid ResponsiblePersonId { get; set; }
    public Guid AccountantId { get; set; }
}